src/accessors.d

```diff
@@ -22,7 +22,7 @@ struct Write
 
 immutable string GenerateFieldAccessors = `
     mixin GenerateFieldAccessorMethods;
-    mixin(GenerateFieldAccessorMethods);
+    mixin(GenerateFieldAccessorMethods2);
```

Comment:
I think it's better to give the less elegant name to the innermost, least-visible-from-outside method, so I went with duselbaers suggestion.